flo-runner/src/main/java/com/spotify/flo/context/Logging.java

```diff
@@ -78,8 +86,22 @@ void failedValue(TaskId taskId, Throwable valueError, Duration elapsed) {
     }
   }
 
-  void complete(TaskId taskId, Duration elapsed) {
+  void complete(TaskInfo taskInfo, Duration elapsed) {
     LOG.info("Total time {}", formatDurationHMS(elapsed.toMillis()));
+
+    LOG.info("Executed {} out of {} tasks:", statuses.size(), PrintUtils.tree(taskInfo).size());
+    PrintUtils.traverseTree(taskInfo, (taskId, s) -> {
+      final Status status = statuses.get(taskId);
+      final String statusLine;
+      if (status == null) {
+        statusLine = "Pending";
+      } else if (status.success()) {
+        statusLine = "Success";
+      } else {
+        statusLine = "Failure: " + status.failure().map(Throwable::toString).orElse("Unknown Error");
+      }
+      LOG.info("{}: {}", s, statusLine);
```

Comment:
Maybe the summary should be a single multi-line log message?

Reply:
Makes sense. Although it would be multiple lines either way in stackdriver.

Reply:
A single entry with multiple lines might be easer for users to view & copy-paste etc.